$NetBSD: patch-CMakeLists.txt,v 1.8 2022/10/15 19:56:55 triaxx Exp $

For pkgsrc. Install config files to examples directory.
Find lua more reliably.

--- CMakeLists.txt.orig	2021-11-01 14:33:30.000000000 +0000
+++ CMakeLists.txt
@@ -155,7 +155,7 @@ ELSE(ENABLE_LUAJIT MATCHES "ON")
 			"include/lua53"
 			"include/lua"
 			ROOT ${LUA_ROOT}
-			MODULES lua53
+			MODULES lua-5.3
 			OPTIONAL)
 
 	IF (NOT WITH_LIBLUA)
@@ -168,7 +168,7 @@ ELSE(ENABLE_LUAJIT MATCHES "ON")
 				"include/lua54"
 				"include/lua"
 				ROOT ${LUA_ROOT}
-				MODULES lua54
+				MODULES lua-5.4
 				OPTIONAL)
 		IF (NOT WITH_LIBLUA)
 			ProcessPackage(LIBLUA LIBRARY	"lua"
@@ -180,7 +180,7 @@ ELSE(ENABLE_LUAJIT MATCHES "ON")
 					"include/lua52"
 					"include/lua"
 					ROOT ${LUA_ROOT}
-					MODULES lua52
+					MODULES lua-5.2
 					OPTIONAL)
 
 			IF (NOT WITH_LIBLUA)
@@ -192,7 +192,7 @@ ELSE(ENABLE_LUAJIT MATCHES "ON")
 						"include/lua51"
 						"include/lua"
 						ROOT ${LUA_ROOT}
-						MODULES lua51)
+						MODULES lua-5.1)
 			ENDIF()
 		ENDIF()
 	ENDIF()
@@ -679,9 +679,9 @@ FILE(GLOB_RECURSE CONF_FILES RELATIVE "$
 	"${CMAKE_SOURCE_DIR}/conf/*.inc" )
 FOREACH(CONF_FILE ${CONF_FILES})
 	GET_FILENAME_COMPONENT(_rp ${CONF_FILE} PATH)
-	INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${CONFDIR}/${_rp})")
+	INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${EXAMPLESDIR}/${_rp})")
 	INSTALL(FILES "${CMAKE_CURRENT_SOURCE_DIR}/conf/${CONF_FILE}"
-		DESTINATION ${CONFDIR}/${_rp})
+		DESTINATION ${EXAMPLESDIR}/${_rp})
 ENDFOREACH(CONF_FILE)
 
 # Lua plugins
